common/persistence/elasticsearch/esVisibilityStore.go

```diff
@@ -772,6 +781,10 @@ func createVisibilityMessage(
 		Version:     common.Int64Ptr(taskID),
 		Fields:      fields,
 	}
+
```

Comment:
How about add a default branch in the switch ? Also I think we should just panic in this case